spec/support/schema.graphql

```diff
@@ -0,0 +1,95 @@
+type Mutation {
+  createOrganization(data: NewOrganization!): Organization
+  deleteAccount(password: String!): Boolean
+  deleteOrganization(id: ID!): Boolean
+  saveAccount(data: UserChangeset!, password: String!): User
+  saveOrganization(id: ID!, data: OrganizationChangeset!): Organization
+}
```

Comment:
I changed the `Boolean!` to `Boolean`, as this is easier to handle validation and authorization errors (just return null if there are errors). Otherwise these are the same.